CONFIGURATION.md

```diff
@@ -199,3 +240,42 @@ Here, the `-git--([\w]+)--` pattern will be replaced with the `X-Tag` value, eff
 As this will lead to a large number of temporary actions in OpenWhisk, we also introduce a `hlx undeploy` command that clears a temporary deployment made earlier. This can be run at the end of every CI job.
 
 `hlx test` and `hlx perf` should use the new temporary strains when running in CI.
+
+## Discussion: layout
+
+During the 4/2018 hackathon we also discussed the layout of the project. it was decided that:
+
+- templates and pre functions will be in `./src`
+- client side javascript **must not** go into `./src`. 
+- we will provide a react example, and put the sources in `./react`
+- all static files will go to `htdocs`
+- all additional openwhisk actions will go to `htdocs/cgi-bin`
+
+## Discussion: the `code` property
+
+- the `code` property points to the code repository
```

Comment:
+ owner, and ref.

Reply:
the `code` property of a strain is either a giturl string or giturl object (but not an action name)

Reply:
That's a better way to phrase it.